src/components/user-item.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import {Text, View} from 'react-native';
+import {UserNode} from '../utils/models';
+
+const phoneMask = (value: string) => {
+  if (!value) {
+    return '';
+  }
+  value = value.replace(/\D/g, '');
+  value = value.replace(/(\d{2})(\d)/, '($1) $2');
+  value = value.replace(/(\d)(\d{4})$/, '$1-$2');
+  return value;
```

Comment:
aqui vc está alterando o argumento `value` que a função recebe, n daria pra evitar isso e utilizar os replaces em sequencia?

```suggestion
  return value.replace(/\D/g, '').replace(/(\d{2})(\d)/, '($1) $2').replace(/(\d)(\d{4})$/, '$1-$2');
```

assim ele vai executar cada replace com o resultado da ultima execução e n altera o value